financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessViewModel.kt

```diff
@@ -47,32 +38,12 @@ internal class SuccessViewModel @Inject constructor(
         suspend {
             val manifest = getManifest()
             val accounts = getCachedAccounts()
-            val saveToLinkWithStripeSucceeded: Boolean? = saveToLinkWithStripeSucceeded.get()
             SuccessState.Payload(
                 skipSuccessPane = manifest.skipSuccessPane ?: false,
-                successMessage = getSuccessMessages(
-                    isLinkWithStripe = manifest.isLinkWithStripe,
-                    isNetworkingUserFlow = manifest.isNetworkingUserFlow,
-                    saveToLinkWithStripeSucceeded = saveToLinkWithStripeSucceeded,
-                    businessName = manifest.businessName,
-                    connectedAccountName = manifest.connectedAccountName,
-                    count = accounts.size
-                ),
-                merchantDataAccess = MerchantDataAccessModel(
-                    businessName = manifest.businessName,
-                    permissions = manifest.permissions,
-                    isStripeDirect = manifest.isStripeDirect ?: false,
-                    dataPolicyUrl = FinancialConnectionsUrlResolver.getDataPolicyUrl(manifest)
-                ),
-                accounts = accounts,
-                institution = manifest.activeInstitution!!,
-                businessName = manifest.businessName,
-                disconnectUrl = FinancialConnectionsUrlResolver.getDisconnectUrl(manifest),
-                accountFailedToLinkMessage = getFailedToLinkMessage(
-                    businessName = manifest.businessName,
-                    saveToLinkWithStripeSucceeded = saveToLinkWithStripeSucceeded,
-                    count = accounts.size
-                )
+                accountsCount = accounts.size,
+                customSuccessMessage = null,
+                // We just want to use the business name if the feature is enabled in the manifest.
+                businessName = manifest.businessName?.takeIf { manifest.useContinueWithMerchantText() },
```

Comment:
Can we move this into a computed property on the manifest?

Reply:
this is just used on the success screen (the business name is used as-is in the rest of the flow)